prometheus.yml

```diff
@@ -0,0 +1,13 @@
+global:
```

Comment:
is this file necessary here?
It's a config file for prometheus, not for the engine.

Reply:
Agree, this is not absolutely necessary but really good to have especially if we want in dev to run some analytics and maybe the dev script should start it too and in that case, it would be useful.

What do you think is the best?
- Keep it in order to have it in dev too
- Make it an example > `prometheus.example.yml`
- Move it to another config repo like https://github.com/mesg-foundation/testnet-ansible
- Add it to the documentation
- Just get rid of it

Reply:
Move it to ansible repo ;) the `testnet-ansible` should also be renamed to just `ansible`.

Reply:
done